styleguide/source/assets/scss/02-molecules/_article-stub.scss

```diff
@@ -169,6 +169,14 @@
       @include type($myriad-pro, $font-weight-bold);
     }
   }
+
+  &--add-padding {
```

Comment:
A new one off class is not needed. You can instead nest it underneath `ama__category__article-stub-three-up` and then `ama__article-stub`

Reply:
Done. I was worried that this would effect other instances of "stub-three-up", but I checked and it looks good.